IceCubesApp/Resources/Localization/Localizable.xcstrings

```diff
@@ -41668,6 +41668,16 @@
         }
       }
     },
+    "settings.content.require-alt-text" : {
```

Comment:
Unfortunately, due to how String catalog works, the default Enligh has to be added to all languages.

Reply:
Thanks, I thought it used a fallback. I have added the English translation to all the other languages now.